source/javascripts/hooks/useIsTruncated.ts

```diff
@@ -24,8 +24,7 @@ const useIsTruncated = (ref: RefObject<HTMLElement>) => {
         observer.unobserve(refCurrent);
       }
     };
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [ref.current]);
+  }, [ref]);
```

Comment:
I know I made this hook, but I'd like to get rid of if.
Maybe simply inlining `element.scrollWidth > element.clientWidth` in the render method where it is used?